content/java/concepts/math-methods/terms/random/random.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.random()'
+Description: 'Returns a pseudorandom double that is 0.0 or greater, and less than 1.0.'
```

Comment:
Consider this sentence instead:

Returns a pseudorandom double greater than or equal to 0.0 and less than 1.0.